src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -451,7 +457,10 @@ namespace dd
           }
         else // classification
```

Comment:
Comment is outdated, should be `classification / regression` ?

Same thing l.677